src/panels/config/users/ha-user-editor.ts

```diff
@@ -154,13 +171,23 @@ class HaUserEditor extends LitElement {
       showSaveSuccessToast(this, this.hass!);
       fireEvent(this, "reload-users");
     } catch (err) {
-      alert(`Group update failed: ${err.message}`);
+      alert(
+        `${this.hass!.localize(
+          "ui.panel.config.users.editor.group_update_failed"
+        )} ${err.message}`
+      );
       selectEl.value = this.user!.group_ids[0];
     }
   }
 
   private async _deleteUser(ev): Promise<void> {
-    if (!confirm(`Are you sure you want to delete ${this._name}`)) {
+    if (
+      !confirm(
+        `${this.hass!.localize(
+          "ui.panel.config.users.editor.confirm_user_deletion"
+        )} ${this._name}`
```

Comment:
Lets give `this._name` as a variable to localize:
```js
${this.hass!.localize("ui.panel.config.users.editor.confirm_user_deletion", "name", this._name)}
```
and add the variable name to the translation: `{name}`

Reply:
Applied it in latest change